### PR TITLE
Task6

### PR DESCRIPTION
**Pull request by YuruMaksau**
Task->done

### CODE REVIEW

**Comment by pahuta** [comment]
Style lint issue. Check `npm run stylelint`

File: Task6/index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Task6</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+<header class="ts6-header">
+`
```
Comment (pahuta) [inline Task6/index.html]: I guess a redundant char "`".

File: Task6/index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    <div class="ts6-social">
```
Comment (pahuta) [inline Task6/index.html]: I guess the class name `ts6-social` doesn't reflect a semantic meaning that this element contains.
In my opinion `ts6-background` is better

File: Task6/index.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    </div>
+    <div class="ts6-signboard">Social Sprites Icons</div>
+</header>
+
+<main>
+
+  <div class="ts6-layout">
+      <div class="ts6-field">
+            <button class="ts6-button__field ts6-button-large ts6-pink">Pink Button</button>
+            <button class="ts6-button__field ts6-button-middle ts6-green">Green Button</button>
```
Comment (pahuta) [inline Task6/index.html]: `ts6-button-large` - it's an element
`ts6-button-middle` - it's an element too
???
How about BEM?
I mean: element - `ts6-button` and modifier - `ts6-button_large`

Comment (pahuta) [reply Task6/index.html]: Let's look at the class `ts6-button__field`:
`ts6-button` - it's a block
`__field` - it's an element

But I can't find block `ts6-button` into your code!

Comment (pahuta) [reply Task6/index.html]: `ts6-pink` - what is it? It's a block? I don't think so. Looks like it should be a modifier.

File: Task6/css/header.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
+body {
+    width: 1200px;
+    margin: 0 auto;
+}
```
Comment (pahuta) [inline Task6/css/header.css]: What do styles for `body` do in `header.css` file?

File: Task6/css/header.css [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+    background: url("../image/spice-social-gadget-sprite.png") no-repeat;
+    background-size: 1100% 300%;
+    width: 45px;
+    height: 45px;
+    margin: 12px;
+    border-radius: 5px;
+    background-position-y: 50%;
+}
+
+.ts6-social-line__icon_icon-1 {
```
Comment (pahuta) [inline Task6/css/header.css]: Let's move all "icons" CSS rules to a separate file. In future these icons can be applied anywhere not only in `header`, so make them free of context.

File: Task6/css/button.css [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    font-weight: bold;
+    margin: 0 5px 8px 0;
+    cursor: pointer;
+    outline: none;
+}
+
+.ts6-button__field:hover {
+    background: #a6a3a5;
+    color: #33292e
+}
```
Comment (pahuta) [inline Task6/css/button.css]: You can achieve more beautiful result via using CSS function like `brightness()`.

File: Task6/css/button.css [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    background: #fc9b00;
+}
+
+.ts6-yellow {
+    background: #feea00;
+}
+
+.ts6-dark {
+    background: #393932;
+}
```
Comment (pahuta) [inline Task6/css/button.css]: Just for improve your experience: use CSS variables for all listed above colors (blue, red, yellow, dark, ...).

File: Task6/index.html [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+            <button class="ts6-button__field ts6-button-middle ts6-red">Red Button</button>
+      </div>
+
+      <div class="ts6-field">
+            <button class="ts6-button__field ts6-button-middle ts6-orange">Orange Button</button>
+            <button class="ts6-button__field ts6-button-middle ts6-yellow">Yellow Button</button>
+      </div>
+  </div>
+
+  <div class="ts6-layout ts6-dark">
```
Comment (pahuta) [inline Task6/index.html]: `ts6-dark` - Looks like it should be a modifier for `ts6-layout`. Isn't it?
Also, in my opinion class name `layout` doesn't reflect a semantic meaning that this element contains.

File: Task6/index.html (hunk 1 above)
Comment (YuruMaksau) [reply Task6/index.html]: Fixed.

File: Task6/index.html (hunk 2 above)
Comment (YuruMaksau) [reply Task6/index.html]: Done.

File: Task6/css/header.css (hunk 4 above)
Comment (YuruMaksau) [reply Task6/css/header.css]: Fixed.

File: Task6/index.html (hunk 3 above)
Comment (YuruMaksau) [reply Task6/index.html]: All comments were fixed.

File: Task6/css/header.css (hunk 5 above)
Comment (YuruMaksau) [reply Task6/css/header.css]: Done.

File: Task6/index.html (hunk 8 above)
Comment (YuruMaksau) [reply Task6/index.html]: Done.

File: Task6/css/button.css (hunk 6 above)
Comment (YuruMaksau) [reply Task6/css/button.css]: Done.

File: Task6/css/button.css (hunk 7 above)
Comment (YuruMaksau) [reply Task6/css/button.css]: Done.

**Comment by YuruMaksau** [comment]
All comments were fixed.

**Pull request merged**